roadmap/implementers-guide/src/runtime/disputes2.md

```diff
@@ -0,0 +1,64 @@
+# Disputes
+
+## Motivation
+
+A reconstructed PoV can be verified with the defined verification code, that is valid during the session the block was included.
+If the block is invalid and there exists at least one backing vote and one disputing vote, a dispute exists,
+which was either detected during the backing stage by a backing checker
+or, made it through the backing stage, but was then found invalid by an approval checker which
+propagate his vote.
+
+At this point the set of backing validators can not be trusted (since they voted for the block despite something being
+fishy at the very least). On the other hand, one must also consider, the validator that blew the whistle has ulterior motives
+to do so (i.e. it is controlled by a third party and wants to incur damage to itself).
+In either way, there are malicious validators around.
+As a consequence, all validators at the time of block backing, are being notified via broadcast of
+the first challenging vote.
+Validators that backed the candidate implicitly count as votes. Those validators are allowed to cast
+a regular vote (a non-backing vote) as well, but it is generally not in their interest to vote both sides, since that would
+advance the progress towards super majority either way and have their bonds slashed.
```

Comment:
It would be helpful to define the terms `quorum` and `supermajority`.
```suggestion
advance the progress towards supermajority either way and have their bonds slashed.
```